api/agent/protocol/json_test.go

```diff
@@ -138,8 +142,8 @@ func TestJSONProtocolDumpJSONRequestWithQuery(t *testing.T) {
 	if err != nil {
 		t.Error(err.Error())
 	}
-	if incomingReq.QueryParameters != req.URL.RawQuery {
-		t.Errorf("Request query string assertion mismatch: expected: %s, got %s",
-			req.URL.RawQuery, incomingReq.QueryParameters)
-	}
+	// if incomingReq.QueryParameters != req.URL.RawQuery {
```

Comment:
this should be changed to request URL assertion

Reply:
Updating tests (and fixing circle errors).